docs/api/tooling/meta-cli.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 2
+sidebar_label: Meta CLI
+---
+
+# Gear Meta CLI
+
+CLI tool to encode/decode payloads and work with .meta.wasm files.
+
+## Installation
+
+```sh
+npm install -g @gear-js/gear-meta
+```
+
+or
+
+```sh
+yarn global add @gear-js/gear-meta
+```
+
+## Usage
+
+### Full list of commmands
+
+```sh
+gear-meta --help
+```
+
+### Available commands
+
+<b>decode</b> - _Decode payload from hex_
+<b>encode</b> - _Encode payload to hex_
+<b>meta</b> - _Display metadata from .meta.wasm_
+<b>type</b> - _Display type structure for particular type from .meta.wasm_
+
+You can simply run these commands and you will be prompted to enter the neccessary data. Or you can specify data through options:
+
+<b>-t, --type</b> - _Type to encode or decode the payload. If it will not specified you can select it later_
+<b>-m, --meta</b> - _Path to .meta.wasm file with program's metadata_
+<b>-o --output</b> - _Output json file. If it doesn't exist it will be created_
+<b>-j --payloadFromJson</b> - _If need to take the payload from json_
```

Comment:
It works bad, see: https://deploy-preview-205--competent-kepler-ce8c2a.netlify.app/api/tooling/meta-cli#available-commands

You may use asterisks for `**bold**`.

Reply:
fixed